js/ui/windowManager.js

```diff
@@ -1211,6 +1403,22 @@ const WindowManager = new Lang.Class({
         }
     },
 
+    _rotateInCompleted: function(actor) {
+        if (this._removeEffect(this._rotateInActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.opacity = 255;
+            actor.rotation_angle_x = 0;
+        }
+    },
+
+    _rotateOutCompleted: function(actor) {
+        if (this._removeEffect(this._rotateOutActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.hide();
```

Comment:
This isn't implemented yet.

I'm not actually sure about the best way to do that. I suppose it could be that when the builder-window is closed, we rotate back to the original application. Either that, or we need some sort of mechanism to tell the shell to go back to the original application and hide builder.

I suspect that's going to be a fair bit of work, so I'd prefer to deal with that in a follow up.